doc/client_lua_api.txt

```diff
@@ -768,12 +768,15 @@ Call these functions only at load time!
     * `nodenames`: e.g. `{"ignore", "group:tree"}` or `"default:dirt"`
     * `search_center` is an optional boolean (default: `false`)
       If true `pos` is also checked for the nodes
-* `minetest.find_nodes_in_area(pos1, pos2, nodenames)`: returns a list of
-  positions.
+* `minetest.find_nodes_in_area(pos1, pos2, nodenames, [grouped])`
+    * `pos1` and `pos2` are the min and max positions of the area to search.
     * `nodenames`: e.g. `{"ignore", "group:tree"}` or `"default:dirt"`
-    * First return value: Table with all node positions
-    * Second return value: Table with the count of each node with the node name
-      as index.
+    * If `grouped` is true the return value is a table indexed by node name
```

Comment:
Can the node name be something like 'group:lava' in this case?

Reply:
No, the names returned are always nodes even if you put in a group.

Reply:
Hmmm, I guess a modder could use `minetest.get_item_group()` if they wanted to apply to all nodes in a certain group.
I have no problems with this then